lib/rules/space-before-function-parentheses.js

```diff
@@ -115,3 +115,25 @@ module.exports = function(context) {
         "FunctionExpression": validateSpacingBeforeParentheses
     };
 };
+
+module.exports.schema = [
```

Comment:
Not really sure this is needed, this this rule is going to go away, but I guess it doesn't hurt.